LED-2500 Fix different price in Cart vs on the PDP in case of discount price

diff --git a/Pricing/Price/RegularPrice.php b/Pricing/Price/RegularPrice.php
index 2c4e332..3ec3f10 100644
--- a/Pricing/Price/RegularPrice.php
+++ b/Pricing/Price/RegularPrice.php
@@ -28,7 +28,10 @@ class RegularPrice extends AbstractPrice implements BasePriceProviderInterface
     {
         if ($this->value === null) {
             $price = $this->product->getPrice();
-            $priceInCurrentCurrency = $this->priceCurrency->convertAndRound($price);
+            // BEGINNING OF PATCH
+            // $priceInCurrentCurrency = $this->priceCurrency->convertAndRound($price);
+            $priceInCurrentCurrency = $this->priceCurrency->convertAndRound($price, null, null, 4);
+            // END OF PATCH
             $this->value = $priceInCurrentCurrency ? (float)$priceInCurrentCurrency : 0;
         }
         return $this->value;
